riscv-slic/src/api.rs

```diff
@@ -79,6 +81,20 @@ pub unsafe fn set_priority<I: crate::InterruptNumber>(interrupt: I, priority: u8
 }
 
 /// Stabilized API for pending a software interrupt on the SLIC.
+///
```

Comment:
Sounds like this should be an `unsafe` function, because of all the human-checked prerequisites.

Reply:
I updated the `mecall-backend` export code so it now makes sure that the `mepc` register is not neglected. In this way, users can now call `pend` safely outside and inside interrupt handlers.